hacspec-scrambledb/scrambledb/src/test_util.rs

```diff
@@ -7,11 +7,13 @@ pub fn generate_plain_table() -> PlainTable {
         vec![
             (
                 String::from("Alice"),
-                hash_to_curve::p256_hash::hash_to_curve(b"TestData1", b"sample_dst").unwrap(),
+                b"TestData1".to_vec(),
+                //hash_to_curve::p256_hash::hash_to_curve(b"TestData1", b"sample_dst").unwrap(),
```

Comment:
What's up with all these comments?

Reply:
I forgot to remove the old commented out code. I've removed the comments now.